src/Trie.mo

```diff
@@ -1,19 +1,11 @@
-/// Functional map
 ///
 /// Functional maps (and sets) whose representation is "canonical", and
```

Comment:
Please start the module with a single liner paragraph. It shows up on the index page and long sentences mess up the layout.

To check run `mo-doc` and then open `docs/index.html` in the browser.

Reply:
Ah, I wasn't always checking that file (just the Trie output file).  Makes sense.  Will do.